src/main/kotlin/com/jetbrains/micropython/devices/WemosD1MiniProDeviceProvider.kt

```diff
@@ -0,0 +1,22 @@
+package com.jetbrains.micropython.devices
+
+import com.jetbrains.micropython.settings.MicroPythonUsbId
+import com.jetbrains.python.packaging.PyRequirement
+
+/**
+ * @author vlan
+ */
+class WemosD1MiniProDeviceProvider : MicroPythonDeviceProvider {
+  override val persistentName: String
+    get() = "WEMOS D1 mini Pro"
+
+  override val usbId: MicroPythonUsbId?
+    get() = MicroPythonUsbId(0x10C4, 0xEA60)
+
+  override val documentationURL: String
+    get() = "https://github.com/vlasovskikh/intellij-micropython/wiki/WEMOS-D1-mini"
```

Comment:
I would prefer having a separate wiki page for WEMOS D1 mini Pro with you listed there as an owner of the device. In case of any device-specific issues people will be able to ask you their questions.